src/Arcus.Testing.Assert/AssertCsv.cs

```diff
@@ -507,11 +507,16 @@ internal CsvDifference(CsvDifferenceKind kind, int expected, int actual)
         internal CsvDifference(CsvDifferenceKind kind, string expected, string actual, int rowNumber)
         {
             _kind = kind;
-            _expected = expected ?? throw new ArgumentNullException(nameof(expected));
-            _actual = actual ?? throw new ArgumentNullException(nameof(actual));
+            _expected = QuoteValueUponSpaces(expected ?? throw new ArgumentNullException(nameof(expected)));
+            _actual = QuoteValueUponSpaces(actual ?? throw new ArgumentNullException(nameof(actual)));
```

Comment:
Added this so that if the value is shown in the test output, the difference in spaces is more clearly shown.